drivers/hmis/app/models/hmis/hud/hmis_service.rb

```diff
@@ -35,9 +35,21 @@ class Hmis::Hud::HmisService < Hmis::Hud::Base
     define_method(hud_field_name) { hud_service&.send(hud_field_name) }
   end
 
+  scope :with_service_type, ->(service_type_id) do
+    where(custom_service_type_id: service_type_id)
+  end
+
   scope :in_service_category, ->(category_id) do
     type_ids = Hmis::Hud::CustomServiceType.where(custom_service_category_id: category_id).pluck(:id)
-    where(custom_service_type_id: type_ids)
+    with_service_type(type_ids)
+  end
+
+  scope :with_project_type, ->(project_types) do
+    joins(:project).merge(Hmis::Hud::Project.with_project_type(project_types))
+  end
```

Comment:
I don't think this will work for services at "WIP" enrollments. Did you try that @ZachCMP ? If you create a new enrollment in the UI, it will be WIP (aka incomplete) initially.

Untested, but I think you can achieve this by doing
```
joins(:enrollment).merge(Hmis::Hud::Enrollment.with_project_type(project_types))
```
because the enrollment `with_project_type` scope already takes the WIP behavior into account.

You can add a similar `with_project` scope to Hmis::Hud::Enrollment and use it in a similar way here.

Reply:
Oh, I didn't realize WIP was a condition that had to be handled here too. I already did most of that work in my latest PR to add more filters to assessments. Would you mind if I merged the two together into one PR?

Reply:
Nope go for it. Sorry I forgot to call it out in the ticket, but it's true across the HMIS app in every scenario. Anywhere we're listing out enrollments or enrollment-related data, WIPs should not be excluded (unless it's something the user is explicitly filtering out).